extra-dev/packages/coq-sniper/coq-sniper.dev+8.13/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "Chantal.Keller@lri.fr"
+homepage: "https://github.com/smtcoq/sniper"
+dev-repo: "git+https://github.com/smtcoq/sniper.git"
+bug-reports: "https://github.com/smtcoq/sniper/issues"
+authors: ["Valentin Blot <Valentin.Blot@inria.fr>"
+          "Louise Dubois de Prsique <Louise.Dubois-de-Prisque@inria.fr>"
+          "Chantal Keller <Chantal.Keller@lri.fr"
+          "Pierre Vial <Pierre.Vial@inria.fr>"
+]
+license: "CECILL-C"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml" {>= "4.07.1" }
+  "coq" {>= "8.13~" & < "8.14~"}
+  "coq-metacoq" {= "1.0~beta2+8.13"}
```

Comment:
Is there a reason to force such a very specific version of MetaCoq? Are you expecting your package to break with the actual 1.0 release of MetaCoq?